Source/WebInspectorUI/UserInterface/Workers/Formatter/JSFormatter.js

```diff
@@ -444,7 +447,8 @@ JSFormatter = class JSFormatter
             }
             if (tokenType === "Punctuator") {
                 if (tokenValue === "*") {
-                    builder.removeLastWhitespace();
+                    if (builder.lastTokenWasWhitespace)
+                        builder.removeLastWhitespace();
```

Comment:
NIT: is this only because of the `console.assert` in `FormatterContentBuilder.prototype.removeLastWhitespace`?  IMO we should consider maybe removing it and renaming to `tryRemoveLastWhitespace` or somethin since it seems much more useful as a "do it if you can" rather than a "you must be in this state"